client/src/components/StudentMain.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useEffect } from "react";
+import QuestionForm from "./QuestionForm.js";
+import { fetchDialogs, fetchMessages, createNewDialog, postQuestion } from "../services/api.js";
+import translateServerErrors from "../services/translateServerErrors.js";
+import ErrorList from "./layout/ErrorList.js";
+
+const StudentMain = (props) => {
+    const dialogId = props.match.params.id
+    const [ dialogs, setDialogs ] = useState([])
+    const [ questions, setQuestions ] = useState([])
+    const [ noDialogMessage, setNoDialogMessage ] = useState("")
+    const [ errors, setErrors ] = useState([])
```

Comment:
no space padding within brackets
https://github.com/airbnb/javascript#whitespace--in-brackets